src/style/_typography.scss

```diff
@@ -0,0 +1,85 @@
+@mixin detail-text {
+  font-size: 16px;
+}
+
+@mixin detail-text--bold {
```

Comment:
I suspect the letter spacing is going to be off on some of these, so they might get updated as we get to actually using these.